automotive/src/main/res/values/strings.xml

```diff
@@ -34,10 +34,10 @@
     <string name="phone_sign_in_button_label">Sign in with phone</string>
     <string name="phone_sign_in_primary_text">Choose phone sign-in method</string>
     <string name="pin_sign_in_button_label">PIN</string>
-    <string name="pin_sign_in_primary_text">Sign in with the PIN code</string>
-    <string name="pin_sign_in_secondary_text">Go to google.com/aural/fast-pair on your phone</string>
+    <string name="pin_sign_in_primary_text">Sign in with PIN code</string>
```

Comment:
I think this could be simply, "Sign in with PIN"

Reply:
Done.